addons/payment_authorize/models/payment.py

```diff
@@ -27,6 +27,12 @@ class PaymentAcquirerAuthorize(models.Model):
     authorize_transaction_key = fields.Char(string='API Transaction Key', required_if_provider='authorize', groups='base.group_user')
     authorize_signature_key = fields.Char(string='API Signature Key', groups='base.group_user', compute="_compute_auth_signature_key", inverse="_inverse_auth_signature_key")
 
+    @api.constrains('payment_flow')
+    def _check_authorize_payment_flow(self):
+        for acquirer in self.filtered(lambda x: x.provider == "authorize"):
+            if acquirer.payment_flow == 'form':
+                raise UserError(_('The "Redirection to the acquirer website" payment flow is deprecated for Authorize.net. Please choose the "Payment From Odoo" flow.'))
```

Comment:
Maybe avoid raising an error for inactive acquirers?

Reply:
Last fix should be enough to filter active acquirer